test/inductor/test_torchinductor.py

```diff
@@ -5535,7 +5535,6 @@ def fn(x, p1, p0):
     "test_cpp_wrapper_dynamic_shapes": ("cpu",),
     "test_cudnn_rnn_dynamic_shapes": ("cuda",),
     "test_grid_sampler_2d_dynamic_shapes": ("cpu", "cuda"),
-    "test_kwargs_dynamic_shapes": ("cpu",),
```

Comment:
My goal here was to enable this test. Fixes other meta tests as well.

Previously, the op was calling `run_fallback_kernel` in `fake_tensor.py` due to the lack of Meta impl, so added that.